addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -226,7 +226,8 @@ class Sanitize {
             // Remove empty blocks in <li>
             if (
                 node.nodeName === 'P' &&
-                node.parentElement.tagName === 'LI'
+                node.parentElement.tagName === 'LI' &&
+                !node.parentElement.classList.contains('nav-item')
```

Comment:
Wouldn't this be useful in prior versions as well? And is there a test for it?

Reply:
It is handled in sanitize from 16.0 onwards only.

> And is there a test for it?

Tests written in the first commits would break without this.

Reply:
I had a look, in 15.0 it is done in `setTag` later it was moved to sanitize from 16.0. I'll create a new PR to fix setTag in `15.0`.